frontend-instructor/src/components/coursePage/ContentItem.jsx

```
@@ -0,0 +1,78 @@
+  return (
+    <button
+      onClick={onSelect}
+      className={`block w-95/100 text-left text-sm px-4 py-2 mx-2 my-1 rounded-md transition-colors ${
+        isSelected
+          ? 'bg-blue-500 text-white shadow-sm'
+          : 'bg-gray-50 text-gray-700 hover:bg-blue-100 hover:text-blue-700'
+      }`}
+    >
+      <div className='flex items-center'>
+        {isVideo && (
+          <svg className='w-4 h-4 mr-2' fill='currentColor' viewBox='0 0 20 20'>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)